README.md

```diff
@@ -290,7 +300,22 @@ This are just examples. `chmod` has a lot of different configurations for differ
 `-c`: Collects files to be compressed<br>
 `-v`: makes tar talk a lot. Verbose output shows you all the files being extracted.<br>
 `-z`: tells tar to decompress the archive using gzip<br>
-`-f`: this must be the last flag of the command, and the tar file must be immediately after. It tells tar the name and path of the compressed file.
+`-f`: this must be the last flag of the command, and the tar file must be immediately after. It tells tar the name and path of the compressed file.<br>
+
+`gzip file1 file2 file3` - used to compress any number of files into a .gz compressed format.<br>
```

Comment:
Mention that this method compresses each of the files separately and not into a single .gz archive.